components/common/typescript.gni

```diff
@@ -137,12 +138,11 @@ template("transpile_web_ui") {
       "--depfile_path=" + rebase_path(depfile),
     ]
 
-    if (defined(invoker.webpack_alias)) {
-      args += [ "--webpack_alias=" + invoker.webpack_alias ]
-    } else {
-      # For explanation of "chromeapp", see:
-      # https://github.com/brave/brave-browser/issues/5587
```

Comment:
Let's move this comment to `components/brave_webtorrent/extension/BUILD.gn`.

Reply:
done